lib/influxer/metrics/metrics.rb

```diff
@@ -1,11 +1,12 @@
 require 'influxer/metrics/relation'
 require 'influxer/metrics/scoping'
 require 'influxer/metrics/fanout'
+require 'active_record'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.